### DISCUSSION TITLE
How to remove colouring of spaces and tabs in VS Code?

### DISCUSSION THREAD

**Discussion by MiltoMiltiadou** [microsoft/vscode-discussions#1585] [Extension Development QnA]
Hello, does anyone know how to remove the colouring of the spaces and tabs in Visual Studio Code? I just installed it and it appears to happen on all themes. If someone can help it will be very useful as I struggle to read my code with so much colour noise.  Image attached.
<img width="108" alt="Untitled" src="https://github.com/user-attachments/assets/ce38b69a-cc96-4c32-81de-262cc6f9f9f4">

**Comment by MiltoMiltiadou** [comment]
I had to unistall the indent-rainbow extension that came by default with the new installation.

**Comment by kolchagov** [comment]
I don't know why is this extension so highly rated, it is offensive and installed without approval. For python fans might work, but for every other language is just gingerbread tastelessness
